tests/setData.spec.ts

```diff
@@ -246,4 +246,33 @@ describe('setData', () => {
 
     expect(wrapper.vm.getResult()).toStrictEqual(`test2: ${expectedResult}`)
   })
+
+  it('should be possible to replace a primitive value with another', async () => {
+    const wrapper = mount(
+      defineComponent({
+        template: '<div />',
+        data() {
+          return {
+            firstArray: [],
+            secondArray: []
+          }
+        }
+      })
+    )
+
+    await wrapper.setData({
+      firstArray: [],
+      secondArray: []
```

Comment:
Maybe use different values than the original ones, like `firstArray: [1, 2], secondArray: [3, 4]`